$OpenBSD: patch-FunctionBar_c,v 1.4 2020/12/08 16:23:17 sthen Exp $

This function is called at the wrong time and will cause use-after-free's,
so we leak for now.

Index: FunctionBar.c
--- FunctionBar.c.orig
+++ FunctionBar.c
@@ -59,14 +59,8 @@ FunctionBar* FunctionBar_new(const char* const* functi
 }
 
 void FunctionBar_delete(FunctionBar* this) {
-   for (int i = 0; i < 15 && this->functions[i]; i++) {
-      free(this->functions[i]);
-   }
    free(this->functions);
    if (!this->staticData) {
-      for (int i = 0; i < this->size; i++) {
-         free(this->keys[i]);
-      }
       free(this->keys);
       free(this->events);
    }
